code/__DEFINES/cargo.dm

```diff
@@ -47,3 +47,17 @@
 #define EXPORT_SOLD 1
 ///Sell the item, but for the love of god, don't delete it, we're handling it in a fancier way.
 #define EXPORT_SOLD_DONT_DELETE 2
+
+
+//At 320 it's 9.8 minutes, at 1400 it's 13.18 minutes,  at 3000 (around gun crates) it's 14.5 minutes, at 8000 (hat crate) 17 minutes, at 9000 (expensive atmos cans) it's 17.3 minutes, and at the 20k crate it's 19.18 minutes.
```

Comment:
Check my math here but is this comment accurate? 200*log(320)^1.2 is 602, so in seconds it's an even 10 minutes and 2 seconds. Same for the 3k estimate, I got 14.87 minutes.

Reply:
You are correct, I probably have not updated the comment values properly.